<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #232323;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="module">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_evryth">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backspsce">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_input">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>80</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>135</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="one_divides">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>135</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">unary_operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>135</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">unary_operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="power_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>135</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">unary_operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="times">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>190</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>190</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>245</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>245</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>245</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>245</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);        
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">operators</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>355</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="equals">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>355</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
color: rgb(255, 255, 255);
background-color: #14446b	;
border: none       
}
                
QPushButton:hover {               
background-color: #2E76CF;                
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_sign">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>355</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>355</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {                
background-color: rgb(24, 24, 24);                
color: rgb(255, 255, 255);               
border: none       
}
                
QPushButton:hover {               
background-color: #666;                
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">digits</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>39</y>
      <width>255</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>8</y>
      <width>255</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="unary_operators"/>
  <buttongroup name="operators"/>
  <buttongroup name="digits"/>
 </buttongroups>
</ui>
